Use PHPDoctor/phpDocumentor to generate documentation.

senky opened the pull request:
Documentation should be part of every bigger project. SN lacks that, and as it will continue to grow, still more and more developers would like to use its API, what is not documented in no way.

This should contain:
1. adding documentation comments to every function, method and class of SN
2. use [PHPDoctor](https://github.com/peej/phpdoctor/) or [PHPDocumentor](http://www.phpdoc.org/) to generate documentation

Comments are more than welcomed, mainly about if you agree with software used (PHPDoctor), or you know about any other, better software.

**WARNING: PR in progress, do not merge it, yet!**

Culprit-CZ commented:
I wanted to use, but Kamahl persuaded me that when we are two, that it is not necessary. 

I know PHPDocumentor. I vote for it.

senky commented:
This is opensource software, not freeware. We will use it. Ok, I do not know phpDocumentor much, but if it is comprendious, I have no problem using it.

Kamahl19 commented:
I have not seen its purpose because all your functions were documented in Czech language or with bad English, which was not understandable. I have nothing against, now it is much more open source then it was before, we use github, there is 3rd developer and we want to open it to world so ok, I agree with using this

senky commented:
Another tip: http://apigen.org/ - looks pretty nice - http://wcdocs.woothemes.com/apidocs/, http://api.nette.org/2.0/

senky committed:
[i/26] adding documentation to root/socialnet/includes/functions.php

Even though diff changes that whole file was replaced, only documentation comments before functions, class and methods have been added.
